### ISSUE TITLE
[Bug] Image alt text is not accessible

### ISSUE THREAD

**Issue by elazar** [duckduckgo/Android#5669]
### Describe the bug

In Chrome, long-tapping on an image (e.g. the one on [this page](https://www.smbc-comics.com/comic/hot-2)) [displays a modal with that image's `alt` text](https://github.com/user-attachments/assets/f08ccfd6-3599-4716-87d2-a01a4565455a).

DuckDuckGo [behaves in a similar but significantly less useful way](https://github.com/user-attachments/assets/ace65cd5-79a3-48ef-a311-0537a1295ce7). Consequently, there's no way (that I've found, at least) to view image `alt` text in it.

Some sites, like https://m.xkcd.com, fill this gap by including the `alt` text in an element on the page that's [collapsed by default](https://github.com/user-attachments/assets/9b27ff3b-e5db-473a-9c19-df5d73e9fb1a) and [can be expanded to view the text](https://github.com/user-attachments/assets/b0004035-6364-43b3-881c-4a912dd6636a).

Unfortunately, many sites do not do this, and leave it up to the user's browser to provide this functionality. I believe this is an important accessibility feature and should be included in DDG.

Given some rough direction on where to start in the codebase to do so, I *might* be able to submit a PR to add this.

In any case, I hope you'll consider adding it.

### How to Reproduce

1. Navigate to a page with an image large enough to interact with that has `alt` text.
2. Long-tap on the image.
3. Observe that the modal that appears does not display the `alt` text.

### Expected behavior

If an image has `alt` text, long-tapping that image should include that `alt` text in the displayed modal.

### Environment

```markdown
- DDG App Version: 5.225.0 (52250000)
- Device: Samsung Galaxy S23 Ulta
- OS: Android 14
```

**Comment by nalcalag** [comment]
Thanks for the report @elazar , this improvement has been reported internally and will be prioritised accordingly. However, since this is not a bug but rather the way it was originally implemented, we can't classify it as a bug fix. For that reason, I’m closing this issue.
